integrations/wp-sweep/wp-sweep.php

```diff
@@ -49,11 +49,19 @@ public function wp_sweep_excluded_termids( $excluded_term_ids ) {
 
 		$excluded_term_ids = array_merge( $excluded_term_ids, $_term_ids );
 
+		$language_taxonomies = PLL()->model->translatable_objects->getIterator();
+		$languages           = PLL()->model->get_languages_list();
+		$lt_term_ids         = array();
+		foreach ( $language_taxonomies as $language_taxonomy ) {
+			foreach ( $languages as $language ) {
+				$lt_term_ids[] = $language->get_tax_prop( $language_taxonomy->get_tax_language(), 'term_id' );
+			}
+		}
+
 		// Add the terms of our languages.
 		$excluded_term_ids = array_merge(
 			$excluded_term_ids,
-			pll_languages_list( array( 'fields' => 'term_id' ) ),
-			pll_languages_list( array( 'fields' => 'tl_term_id' ) )
+			$lt_term_ids
 		);
```

Comment:
You can do much simpler by:
1. Avoiding a direct call to `getIterator()`. It's for PHP, you probably never it.
2. Avoiding an intermediate array ( again ;) )
3. Use `get_tax_prop**s**()` which definitely kills the bullet 1.

```php
		// Add the terms of our languages.
		foreach ( $languages as $language ) {
			$excluded_term_ids = array_merge(
				$excluded_term_ids,
				$language->get_tax_props( 'term_id' );
			);
		}
```